src/base/bittorrent/session.cpp

```diff
@@ -1445,60 +1445,53 @@ void Session::configureNetworkInterfaces(lt::settings_pack &settingsPack)
 
     QStringList endpoints;
     QStringList outgoingInterfaces;
-    const QStringList listeningIPs = getListeningIPs();
     const QString portString = ':' + QString::number(port);
 
-    if (listeningIPs.isEmpty()) {
-        endpoints << (QHostAddress(QHostAddress::AnyIPv4).toString() + portString);
-        endpoints << ('[' + QHostAddress(QHostAddress::AnyIPv6).toString() + ']' + portString);
```

Comment:
This branch isn't needed anymore due to the way "All addresses"/All Ipv4/All IPv6 are returned by getListeningIPs().
So all the code below is shifted one indentation level up.